concent_api/utils/helpers.py

```diff
@@ -46,3 +49,30 @@ def parse_datetime_to_timestamp(date_time: datetime.datetime) -> int:
         return int(date_time.replace(tzinfo = timezone.utc).timestamp())
     else:
         return int(date_time.astimezone(timezone.utc).timestamp())
+
+
+def get_field_from_message(golem_message: message.base.Message, field_name: str) -> str:
+    """
+    Returns field value with given field name nested inside given Golem Message or FrozenDict
+    by checking recursively from top to bottom.
+
+    Returns None if field is not available.
+    """
+
+    def check_task_id(golem_message):
+        assert isinstance(golem_message, (message.base.Message, FrozenDict))
```

Comment:
What about dict? Can golem-messages contain ordinary dicts?

Reply:
From what I have checked they don't at this moment.